sales/js/app.js

```diff
@@ -149,22 +161,53 @@ function renderTableBody(parent, bodyElements, subArrayName){
   }
 }
 
-// Cookies table
-processLocationStats();
-var parentMyElement = document.getElementById('mylistsOfStats');
-var table = document.createElement('table');
-parentMyElement.appendChild(table);
+function renderTables(){
+  // Cookies table
+  processLocationStats();
+  var parentMyElement = document.getElementById('mylistsOfStats');
+  var table = document.createElement('table');
+  parentMyElement.appendChild(table);
 
-renderTableHeader(table, providingHeaders('Daily Location Total','Store Name'));
-renderTableBody(table, stores, 'renderStatsAsRow');
-renderTableFooter(table, providingSumRow());
+  renderTableHeader(table,
+    providingHeaders('Daily Location Total','Store Name'), 'Cookie Sales'
+  );
+  renderTableBody(table, stores, 'renderStatsAsRow');
+  renderTableFooter(table, providingSumRow('locationsStats', 'totalCookiesSold'));
+
+  // Tossers Table
+  processTossersNeed();
+  var table2 = document.createElement('table');
+  parentMyElement.appendChild(table2);
+  renderTableHeader(table2,
+    providingHeaders('Total Tosser Hours','Store Name'), 'Tossers Per Hour'
+  );
+  renderTableBody(table2, stores, 'renderTossersAsRow');
+  renderTableFooter(table2, providingSumRow('tossersNeed', 'totalTossersNeeded', 'Sub Total People needed'));
+}
 
-// Tossers Table
+function unloadElementWithId(elementId){
+  var tableToUnload = document.getElementById(elementId);
+  tableToUnload.textContent = '';
+}
 
-processTossersNeed();
-var table2 = document.createElement('table');
-parentMyElement.appendChild(table2);
-renderTableHeader(table2, providingHeaders('Total Tosser Hours','Store Name'));
-renderTableBody(table2, stores, 'renderTossersAsRow');
+renderTables();
+function newStoreHandler (event) {
```

Comment:
I also created my event handler as a separate function and then called it as part of my listener! Great job!